sky/build/zip_directory.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+#
+# Copyright 2016 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+'''Zips up an entire directory.'''
```

Comment:
Maybe we could use this to solve https://github.com/flutter/flutter/issues/1707 :D